<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSettingsWidget</class>
 <widget class="QWidget" name="SPRSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabSettingsWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_10">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/SettingsDevice.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/SettingsDevice.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Устройство</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="SPRSettingsMainWidget" name="wSettingsMainWidget" native="true">
               <property name="minimumSize">
                <size>
                 <width>391</width>
                 <height>454</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>451</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpectrumZones">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Spector_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Spector_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Области спектра</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SPRZonesRangesTableWidget3" name="wSpectrumZonesWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Formulas_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Formulas_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Аналитический параметр</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SPRSettingsFormulaWidget" name="wSettingsFormulaWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Algoritm_Sittings_Porogs_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Algoritm_Sittings_Porogs_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Пороги</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="SPRPorogsWidget2" name="wSettingsPorogsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Ims_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Ims_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>ИМ</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SPRSettingsIMSWidget" name="wSettingsIMSWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_SeparationControl_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_SeparationControl_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Контроль</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SPRSettingsControlWidget2" name="wSettingsControl" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Test">
      <attribute name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Rentgen_Menu_tab_1.png</normaloff>:/icons/settings/icons/AlgoripmSerrings_tabs/Alroritm_Settings_Rentgen_Menu_tab_1.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Измерительный тракт</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SPRSettingsRentgenWidget" name="wSettingsRentgen" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bCancel">
        <property name="text">
         <string>Отменить</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/b_Cancel.png</normaloff>:/icons/icons/b_Cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSave">
        <property name="text">
         <string>Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/b_Save.png</normaloff>:/icons/icons/b_Save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSaveAs">
        <property name="text">
         <string>Сохранить как...</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/filenew.png</normaloff>:/icons/icons/filenew.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>509</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SPRSettingsMainWidget</class>
   <extends>QWidget</extends>
   <header>sprsettingsmainwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRZonesRangesTableWidget3</class>
   <extends>QWidget</extends>
   <header>sprzonesrangestablewidget3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRPorogsWidget2</class>
   <extends>QWidget</extends>
   <header>sprporogswidget2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSettingsFormulaWidget</class>
   <extends>QWidget</extends>
   <header>sprsettingsformulawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSettingsRentgenWidget</class>
   <extends>QWidget</extends>
   <header>sprsettingsrentgenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSettingsIMSWidget</class>
   <extends>QWidget</extends>
   <header>sprsettingsimswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSettingsControlWidget2</class>
   <extends>QWidget</extends>
   <header>sprsettingscontrolwidget2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
